ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -79,6 +79,7 @@ descriptor:
   ## TODO: make it so gas only has one meaning
   gas: "Process or component driven by natural gas (as to distinguish from electric)."
   glycol: "Liquid mixture consisting of glycol and water "
+  group: "Type of light dimming command point."  # Added 25/08/20
```

Comment:
This definition is not sufficient; it does not describe what a group is. Is it lighting specific? Does 'group' mean what someone would naively think it means?